### PR TITLE
Feature/support style rebase

### PR DESCRIPTION
**Pull request by hectoruch**
Rebase Feature/support style
Check and close the other one.

### CODE REVIEW

File: app/src/reducers/appearance.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const initialState = { menuVisible: false, supportModalVisible: false };
```
Comment (tiagojsag) [inline app/src/reducers/appearance.js]: remove this file

File: app/src/components/Map.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import Timebar from '../containers/Map/Timebar';
 import Modal from './Shared/Modal';
 import Share from '../containers/Map/Share';
 import NoLogin from '../containers/Map/NoLogin';
-import MapFooter from './Map/MapFooter';
```
Comment (tiagojsag) [inline app/src/components/Map.jsx]: delete MapFooter.jsx

File: app/src/actions/appearence.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { SET_VISIBLE_MENU } from '../actions';
```
Comment (tiagojsag) [inline app/src/actions/appearence.js]: delete this file

**Comment by tiagojsag** [comment]
make sure you addressed all comments from https://github.com/Vizzuality/GlobalFishingWatch/pull/163

**Comment by andresgnlez** [comment]
@tiagojsag is OK for you merge this?

**Comment by tiagojsag** [comment]
@andresgnlez I saw some comments you placed on #163 . If those are already fixed here, yes, this can be merged

File: app/src/components/Map/SupportModal.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import supportModalStyle from '../../../styles/components/map/c-support-modal.scss';
+import layerSupportModalStyle from '../../../styles/components/c-layer-back.scss';
+
+class SupportModal extends Component {
+
+  render() {
+    return (<div>
+      <div
+        className={[layerSupportModalStyle['c-layer-back'],
+        layerSupportModalStyle['-support-modal']].join(' ')}
```
Comment (andresgnlez) [inline app/src/components/Map/SupportModal.jsx]: https://github.com/Vizzuality/GlobalFishingWatch/pull/163/files/b5aaa26c24af2158bbdef72b99807fb399b4cd08#r75823078

File: app/src/components/ContactUs/SupportForm.jsx [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
     this.state = {
       url: window.location,
       submitted: false,
-      showFormResponse: false
+      showFormResponse: false,
+      classSelect: '',
+      disabledOption: false
     };
 
     this.handleFormSubmit = this.handleFormSubmit.bind(this);
```
Comment (andresgnlez) [inline app/src/components/ContactUs/SupportForm.jsx]: Avoid declaring unnecessary variables for binding, instead use arrow functions.

File: app/styles/components/map/c-support-modal.scss [hunk 6]
```diff
@@ -0,0 +1,38 @@
+@import '../../_settings.scss';
+
+.c-support-modal {
+  background-color: rgba($color-2, .9);
```
Comment (tiagojsag) [inline app/styles/components/map/c-support-modal.scss]: remove the alpha. the modal background is not semi-transparent

File: app/src/components/ContactUs/SupportForm.jsx [hunk 7]
```diff
@@ -49,73 +59,85 @@ class SupportForm extends Component {
...
+              required
+              onChange={this.handleChange}
+            />
+
+            <label htmlFor="support_email">Email</label>
+            <input
+              className={supportFormStyle['input-text']}
+              type="Email"
+              id="support_email"
+              placeholder="email"
```
Comment (tiagojsag) [inline app/src/components/ContactUs/SupportForm.jsx]: Email (upper case E) please keep an eye on this kind of things. This is something that is easy for us to detect when testing after developing.

File: app/src/components/ContactUs/SupportForm.jsx [hunk 8]
```diff
@@ -49,73 +59,85 @@ class SupportForm extends Component {
...
+            <label htmlFor="support_type">Type</label>
+            <div className={supportFormStyle['select-container']}>
+              <select
+                id="support_type"
+                onChange={this.handleChange}
+                name="selectCompany"
+                className={supportFormStyle[this.state.classSelect]}
+                required
+              >
+                <option disabled={this.state.disabledOption}>Select an ...</option>
```
Comment (tiagojsag) [inline app/src/components/ContactUs/SupportForm.jsx]: Select an what???

**Comment by tiagojsag** [comment]
- [The success and error messages are not styled ] 
- [email placeholder with wrong casing]
- [Incorrect select placeholder]
- [Support form not working]
- [Select has incorrect styling (FF at least, test other browsers)]

**Comment by tiagojsag** [comment]
@hectoruch I already fixed some of the issues you had in your code, but we should try to make sure that, in the future, this kind of issues does not make it into CR, ok?

File: app/styles/components/map/c-support-modal.scss (hunk 6 above)
Comment (hectoruch) [inline app/styles/components/map/c-support-modal.scss]: It has opacity on zepplin.

**Comment by tiagojsag** [comment]
the select styling is still broken on chrome on windows

**Comment by tiagojsag** [comment]
also broken on edge win 10

**Comment by tiagojsag** [comment]
The form modal should not be transparent

**Comment by tiagojsag** [comment]
the name and email fields are not prepopulated

**Comment by hectoruch** [comment]
The form modal has opacity on zepplin design @tiagojsag

**Comment by tiagojsag** [comment]
ah ok, my bad

**Comment by hectoruch** [comment]
About "the name and email fields are not prepopulated" I tested that on all browsers and that work fine.
Is a silly question but are you sign up?

**Comment by tiagojsag** [comment]
looked into it, seems to be a result of another bug

**Comment by hectoruch** [comment]
Rebase move to https://github.com/Vizzuality/GlobalFishingWatch/pull/180

**Pull request closed without merging**